modules/admin-ui-picard/app/src/reducers/eventDeatilsReducers.js

```diff
@@ -18,10 +20,46 @@ const initialState = {
     eventId: ""
 }
 
+const createPolicy = (role) => {
+    return {
+        role: role,
+        read: false,
+        write: false,
+        actions: []
+    };
+};
+
 // Reducer for event details
 const eventDetails = (state=initialState, action) => {
     const { type, payload } = action;
     switch (type) {
+        case LOAD_EVENT_POLICIES_SUCCESS: {
+            const { accessPolicies } = payload;
+            let policies = [];
+            if(!!accessPolicies.episode_access){
```

Comment:
Please move this code to the thunk where you fetching the episodes access policies. The redux state should only contain the actual state changes.